scripts/README.md

```diff
@@ -241,14 +244,20 @@ This is a helper script for Knative release scripts. To use it:
    - `RELEASE_GCS_BUCKET`: contains the GCS bucket name to store the manifests if
      `--release-gcs` was passed, otherwise the default value `knative-nightly/<repo>`
      will be used. It is empty if `--publish` was not passed.
+   - `BUILD_COMMIT_HASH`: the commit short hash for the current repo. If the current
+     git tree is dirty, it will have `-dirty` appended to it.
```

Comment:
should this be `--dirty`?

Reply:
No. It's a suffix of a string, not a flag.